src/Modal.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import ReactModal from 'react-modal';
+
+let rootSelector;
+
+export const setAppElement = (selector) => {
+    rootSelector = selector;
+    ReactModal.setAppElement(selector);
+};
+
+export class Modal extends Component {
+    rootElement = null;
+    previousScrollValue = null;
+    prevStyleValues = {};
+
+    componentDidMount() {
+        const { isOpen } = this.props;
+
+        this.rootElement = document.querySelector(rootSelector);
+
+        if (isOpen) {
+            const newStyleValues = this.computeElementValues();
+
+            this.rootElement.style.top = newStyleValues.top;
+            this.rootElement.style.left = newStyleValues.left;
+            this.rootElement.style.right = newStyleValues.right;
+            this.rootElement.style.position = newStyleValues.position;
```

Comment:
Should we also have a function that receives an object and updates `this.rootElement` styles? We are doing this here and at componentDidUpdate

Reply:
Yes, thanks :+1:  Just did that! Called it `updateRootElement` and also removed the `bottom` property from where it was being saved since we don't use that anywhere.